pkg/controller/sharedinfo/shared_info.go

```diff
@@ -17,6 +17,7 @@ type SharedInfo struct {
 
 	NetworkConfig     *configv1.Network
 	OperatorConfigMap *corev1.ConfigMap
+	OperatorNsxSecret *corev1.Secret
```

Comment:
Should we also add lb secret into the shared info?

Reply:
Resources in shared_info are shared between controllers. As we don't have a controller using lb secret, I think we don't need add it there.